app/src/main/java/me/hufman/androidautoidrive/phoneui/viewmodels/PermissionsModel.kt

```diff
@@ -39,6 +43,9 @@ class PermissionsModel(private val notificationListenerState: LiveData<Boolean>,
 	private val _spotifyErrorHint = MutableLiveData<Context.() -> String> { "" }
 	val spotifyHint: LiveData<Context.() -> String> = _spotifyErrorHint
 
+	private val _isSpotifyWebApiAuthorized = MutableLiveData(false)
+	val isSpotifyWebApiAuthorized = _isSpotifyWebApiAuthorized as LiveData<Boolean>
```

Comment:
Use `val isSpotifyWebApiAuthorized: LiveData<Boolean> = _isSpotifyWebApiAuthorized` for slightly higher code coverage, I think the `as` casting counts as a branch for some reason and codecov flags it as yellow :(

Reply:
Oh didn't realize that, updated to your suggestion!